README.md

```diff
@@ -178,12 +181,15 @@ JVM parameters: e.g. `java -jar -Xms600m -Xmx600m scoold-*.jar`
 
 **Scoold requires a persistent and direct connection to a Para server to function properly.**
 
-The most important configuration properties are:
-- `scoold.para_endpoint` - the URL of the Para server
-- `scoold.para_access_key` - the application identifier of your Para app
-- `scoold.para_secret_key` - the secret key for your Para app
+<!-- really needed here? The why is missing. Can we highlight them in the example below instead? -->
+<!-- remove or keep --> 
+<!--The most important configuration properties are:
+- `scoold.para_endpoint` - Para server URL
+- `scoold.para_access_key` - Para app application identifier
+- `scoold.para_secret_key` - Para app secret key
+-->
```

Comment:
See comments, I think this is not required. para_endpoint, para_access_key, and para_secret_key are in the example below and they are pretty clear. There's some kind of authentication with the host needed. 

Also, the list here explains the para_access_key as application identifier, below access key is used. The word access key should be clear.

I users need this info, I'd go with

```suggestion
Make sure to enter the correct para_access_key, para_secret_key, and para_endpoint for authentication with Para.
```

Reply:
Otherwise, I'd remove this completely.

```suggestion
```

Reply:
Since we've added a new auto-configuration feature recently, this part can be removed completely because Scoold can automatically populate these values if that feature is enabled.